### DISCUSSION TITLE
Codespaces check-in - AMA round-up edition

### DISCUSSION THREAD

**Discussion by otter-computer** [community/community#50940] [Codespaces]
Hey all, it’s another Codespaces check-in! 💥

## Asked us anything 🧠
This week was the [Codespaces AMA](https://github.com/orgs/community/discussions/50829) with project managers `@cmuto09`, `@craiglpeters`, and `@thisiskirsch`. Thanks to everyone that came along and asked a question, or just had a look at what everyone was asking!

### AMA Rundown
First up, from the education community, we had some [questions on speeding up devcontainer build times](https://github.com/orgs/community/discussions/50829#discussioncomment-5407534) and the possibility of sharing containers organization-wide or from a fork.

We had [a question on team collaboration](https://github.com/orgs/community/discussions/50829#discussioncomment-5408544), on the best way to share and collaborate with QA testers without waiting on long build times.

From the community, we had a question on [how to access a Codespace via SSH](https://github.com/orgs/community/discussions/50829#discussioncomment-5407900) for use in your terminal or choice, or from automation.

And finally, from the community, a question on [if it’s possible to restore a Codespace](https://github.com/orgs/community/discussions/50829#discussioncomment-5408000) even after the repository it’s attached to is deleted.

Thanks a lot for joining, everyone! Keep an eye on the community for the next one!

**Comment by danielcregg** [comment]
Am I too late to ask a questions here... if not... How do i connect to my desktop-lite via noVNC without using a password? THanks
